src/core/book-keeper.js

```diff
@@ -1,9 +1,277 @@
 'use strict';
 
-class BookKeeper {
-    constructor () {
+var Util = require('./util');
 
+var moment = require('moment');
+
+var metricsEventEmitter = require('../src/core/event/event-bus.js').EventEmitter;
+
+
+var init = function(EventI) {
+
+    EventI.on('DELETE_OLD_SCHEDULE', function(col, days, cb) {
+
+          var query = {};
+          query[toFetch.date] = {
+              $lt: moment().subtract(days, 'days').toISOString()
```

Comment:
db was changed, use ms form for time instead of ISO string by calling .format('x')